### ISSUE TITLE
Uncertainty Resolver has no recipe

### ISSUE THREAD

**Issue by Prometheus0000** [GTNewHorizons/GT-New-Horizons-Modpack#6217]
#### Which modpack version are you using?
2.0.9.0 RC2
#
#### What did you try to do, and what did you expect to happen?
Check out the recipe for the Tectech Uncertainty Resolver, necessary for the Quantum Computer, and thus progression.
#
#### What happened instead? (Attach screenshots if needed)
It has no recipe.
#
#### What do you suggest instead/what changes do you propose?
Recipe be fixed. Or is it fixed on QF2?

**Comment by chochem** [comment]
Also missing in QF2.

**Comment by chochem** [comment]
I feel like this one and T8 rocket need an urgent or major label or something as they break the entire progression of the pack.

Anyway, I tried to find the cause for a bit and found this commit
https://github.com/Technus/TecTech/commit/b3fa98eebb71c25e3a59ed965ca217f40bd68470
apparently new recipes were added for ParametrizerX, ParametrizerTXT, UncertaintyX. Though I cant make out any problem in the code. I should point out that Parametrizer, ParametrizerX, and ParametrizerTXT all have recipes in game while the for UncertaintyX is also missing.

Sadly Tec was extremely unhelpful on discord, only saying that missing recipe = unfinished. I think he just didnt care though and didnt read most of what I wrote. I am still pretty sure this is just a bug.
